Tests/SpeziOmronTests/SpeziOmronTests.swift

```diff
@@ -183,4 +183,7 @@ final class SpeziOmronTests: XCTestCase {
 }
 
 
-extension MockDevice: OmronHealthDevice {}
+#if compiler(>=6)
+#warning("Use @retroactive annotation instead if spelling out module names once we drop support for <6 toolchains.")
```

Comment:
Nice mechanism to keep track of that. Might also make sense to create a small issue for that.

Reply:
I removed the warning and replaced it with an #if #else that uses `@retroactive` if the new compiler is available.